NYPLCardCreator/Classes/NameAndEmailViewController.swift

```diff
@@ -77,10 +77,23 @@ final class NameAndEmailViewController: FormTableViewController {
       animated: true)
   }
   
+  private func emailIsValid() -> Bool {
```

Comment:
By request, this has been reduced in strictness to only check for an '@' symbol and at least one 'dot' following it.